### PR TITLE
add wordrank in dockerfile

### PR DESCRIPTION
**Pull request by parulsethi**
Fixes https://github.com/RaRe-Technologies/gensim/issues/1457

Used `mpich` instead of `openmpi` to avoid multithreading errors in wordrank.

### CODE REVIEW

**Comment by menshikh-iv** [comment]
About wordrank
```
======================================================================
ERROR: testEnsemble (gensim.test.test_wordrank_wrapper.TestWordrank)
Test ensemble of two embeddings
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/gensim/gensim/test/test_wordrank_wrapper.py", line 37, in setUp
    self.test_model = wordrank.Wordrank.train(self.wr_path, self.corpus_file, self.out_name, iter=6, dump_period=5, period=5, cleanup_files=True)
  File "/gensim/gensim/models/wrappers/wordrank.py", line 161, in train
    output = utils.check_output(args=cmd)
  File "/gensim/gensim/utils.py", line 1177, in check_output
    raise error
CalledProcessError: Command '['mpirun', '-np', '1', '../wordrank', '--beta', '99', '--loss', 'hinge', '--epsilon', '0.75', '--period', '5', '--alpha', '100', '--lrate', '0.001', '--path', 'meta', '--dump_period', '5', '--dump_prefix', 'model', '--dim', '100', '--sgd_num', '100', '--nthread', '8', '--iter', '6', '--reg', '0']' returned non-zero exit status 139
```
```
root@18d3d950726b:/gensim/gensim_dependencies/wordrank# mpirun -np 1 ../wordrank --beta 99 --loss hinge --epsilon 0.75 --period 5 --alpha 100 --lrate 0.001 --path meta --dump_period 5 --dump_prefix model --dim 100 --sgd_num 100 --nthread 8 --iter 6 --reg 0
[proxy:0:0@18d3d950726b] HYDU_create_process (utils/launch/launch.c:75): execvp error on file ../wordrank (Permission denied)
```

**Comment by parulsethi** [comment]
@menshikh-iv Updated my `add_wordrank_in_docker` branch with `develop`. And the `Segmentation fault (signal 11)` error can be reproduced by first running gensim's [test_wordrank_wrapper.py](https://github.com/RaRe-Technologies/gensim/blob/develop/gensim/test/test_wordrank_wrapper.py) (to create testmodel directory containing metadata, vocab.txt etc.) and then running the  
```
mpirun -np 1 ../wordrank --beta 99 --loss hinge --epsilon 0.75 --period 5 --alpha 100 --lrate 0.001 --path meta --dump_period 5 --dump_prefix model --dim 100 --sgd_num 100 --nthread 8 --iter 6 --reg 0
```
  from directory `/gensim/gensim_dependencies/wordrank/testmodel`

File: docker/wordrank_install.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```
Comment (menshikh-iv) [inline docker/wordrank_install.sh]: Please remove this script from PR. You already download it with the wordrank repo. After it, use `awk` or `sed` to replace  `#export CC=icc CXX=icc` to `export CC=gcc CXX=g++`

**Comment by menshikh-iv** [comment]
Also, do several things

1. Add parameter  for `-np` to WordRank `__init__`
2. Remove `cd` call from wrapper and use  absolute path to binary instead of relative  `../wordrank`
3. Add comment & link in wrapper about `ompi+docker` problems.

**Pull request merged**